plugins/pulp_ostree/plugins/distributors/configuration.py

```diff
@@ -83,3 +96,42 @@ def get_repo_relative_path(repo, config):
     else:
         path = repo.repo_id
     return path
+
+
+def _check_for_relative_path_conflicts(repo_id, relative_path):
+    """
+    Check that a relative path does not conflict with existing distributors' relative paths.
+
+    :param repo_id: identifier of repository associated with the path to check
+    :type  repo_id: basestring
+    :param relative_path: relative path of the repository
+    :type  relative_path: basestring
+    :return error_messages: a list of validation errors
+    :rtype: list
+    """
```

Comment:
Since the `relative_path` affects the URL used by client to access content, I think the `_url` in variables is confusing here.  Just my $0.02.